spec/audited_spec_helpers.rb

```diff
@@ -1,5 +1,13 @@
 module AuditedSpecHelpers
 
+  def reset_audited_ignored_attributes
+    Audited.ignored_attributes = %w(lock_version created_at updated_at created_on updated_on)
+  end
+
+  def reset_user_non_audited_fields
```

Comment:
woops. These are leftover from me trying to circumvent issues that arose within the test suite which I mentioned in my commit message.